src/util.rs

```diff
@@ -18,26 +19,36 @@ pub fn nom_input_error_with_kind(input: &[u8], kind: ErrorKind) -> Err<nom::erro
     Err::Error(nom::error::Error::from_error_kind(input, kind))
 }
 
-#[derive(Debug)]
 /// Indexed Color identifiers used for memory cues and tracks.
+#[derive(Debug, PartialEq)]
+#[binread]
```

Comment:
this could be `binrw`

Reply:
Yeah, as before I'd like to get read support finished first, before dealing with serialization. If no changes are needed, `sed -i 's/[binread]/[binrw]/g' -e 's/[br(/[brw(/g' anlz.rs` will probably suffice.

Reply:
Thats what I thought about the not as trivial types, but making this type serializeable is just a matter of changing this one line, which is why I proposed it.

Reply:
https://github.com/Holzhaus/rekordcrate/pull/47/commits/f0d4ad0343e0ef0c57f36ba6f20f884d2403500e